tests/cloud_functions/token_refresh/streams/test_strava.py

```
@@ -0,0 +1,36 @@
+from cloud_functions.token_refresh.streams.strava import strava_update_user_tokens
+from tests.cloud_functions.testing_utils import InMemoryFirestoreClient
+
+
+def test_strava_update_user_tokens():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_strava_update_user_tokens`

Add return type annotation: `None`

(ANN201)